ocean_py/agent/purchase_agent.py

```diff
@@ -0,0 +1,28 @@
+"""
+    PublishAgent - Agent to access the ocean publishing and services aka Brizo
+"""
+
+from squid_py import ServiceAgreement, ServiceTypes
+
+from ocean_py.agent.agent_base import AgentBase
+# from ocean_py import logger
+
+class PurchaseAgent(AgentBase):
+    def __init__(self, ocean, **kwargs):
+        """init a standard ocean agent, with a given DID"""
+        AgentBase.__init__(self, ocean, **kwargs)
+
+
+    def purchase_asset(self, asset, account):
+        """
+        Register an asset with the agent storage server
+        :param asset: asset to purchase
+        :param account: account to use for buying this asset
+        """
```

Comment:
Commnet: what is returned for _success_ and _failure_? It looks like NONE would be passed back up in the event of failure but it might make sense to call that out.